VagrantProvisionCentOS7.sh

```diff
@@ -40,8 +40,18 @@ if [ "${ADDREPOS:-yes}" = "yes" ]; then
     # configure PGDG repository for PostgreSQL 9.5.
     echo "### Add pgdg repo ###" >> CentOS_upgrade.txt
     sudo $HOOT_HOME/scripts/yum/pgdg-repo.sh 9.5
+
+    # configure the devtoolset repository
+#    echo "### Add devtoolset repo ###"
+#    sudo yum install -y centos-release-scl
+#    sudo yum-config-manager --enable rhel-server-rhscl-7-rpms
 fi
 
+# configure the devtoolset repository
+echo "### Add devtoolset repo ###"
+sudo yum install -y centos-release-scl yum-utils
+sudo yum-config-manager --enable rhel-server-rhscl-7-rpms
+
```

Comment:
Once the VMs for Vagrant and AWS are updated with this repository, the first part can be uncommented and the second removed.